src/pages/expenseDetail/loader.ts

```
@@ -0,0 +1,58 @@
+      queryKey: ['userInfo'],
+      queryFn: getUserInfo,
+    });
+    if (!user)
+      return redirect(`/login?returnUrl=/expense-detail/${groupToken}`);
+
+    // 2. 프로필 선택 여부 확인
+    const profiles = await queryClient.ensureQueryData({
+      queryKey: ['profiles', groupToken],
+      queryFn: () => getProfiles(groupToken),
+    });
+    const myProfile = profiles.find((profile) => profile.userId === user.id) ?? null;
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)